test/test_chronology.jl

```diff
@@ -5,8 +5,8 @@ else
 end
 
 function test_chronology()
-    stages = Dict(1 => Stage(template_uc, 24, Dates.Hour(24), 1, c_sys5_uc, GLPK_optimizer,  Dict(0 => Sequential())),
-                2 => Stage(template_ed, 12, Dates.Minute(5), 24, c_sys5_ed, GLPK_optimizer, Dict(1 => Synchronize(24,1), 0 => Sequential()), TimeStatusChange(:ON_ThermalStandard)))
+    stages = Dict(1 => Stage(template_uc, 24, Dates.Hour(24), 1, c_sys5_uc, GLPK_optimizer,  Dict(0 => Consecutive())),
```

Comment:
shouldn't this change to be the new stages formulation?

Reply:
This isn't updated yet